manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.err
+PackageVersion: 6.4.5
+PackageLocale: en-US
+Publisher: Microsoft Corporation

```

```suggestion
Publisher: Microsoft Corporation
PublisherUrl: https://microsoft.com/
PublisherSupportUrl: https://support.microsoft.com/
PrivacyUrl: https://privacy.microsoft.com/privacystatement/
```
Additional metadata to make the manifest more complete